.circleci/verify-release.js

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env node
```

Comment:
Could all of this be done with:

```sh
node -pe 'window={}; document={}; require("react-axe")'
```

Then for ensuring the TS definitions are included, doing:

```sh
npx ts-node -pe 'import axe = require("react-axe")'
```

Reply:
I tried `npx ts-node -pe 'import axe = require("react-axe")'` and it actually didn't do anything. Deleting the types file didn't fail the command at all. Doing `require('react-axe')` didn't work either as `window` and `document` needed to be defined but I couldn't define them myself like I did for the node script as they are strictly typed already. So doing `window = {}` would fail as `.type window` is `declare var window = Window & globalThis`.